I'm sorry, but an uncaught exception occurred.

While loading <'Image' u'images/by_Sysen/Troy/barista1.png'>:
  File "game/script.rpy", line 614, in script call
    credits = ('Backgrounds', 'Uncle Mugen'), ('Sprites', 'Deji'), ('Writing', 'meltycreamcats'), ('Writing', 'margaretcatter'), ('Programming', 'margaretcatter'), ('Programming', 'Arsym'), ('GUI Design', 'Re:Alice') ('Music', 'Kevin MacLeod'), ('Music', 'XXX'), ('SFX', 'Mike Koenig'), ('GFX Wipes', 'Kia')
  File "game/Beta Ending.rpy", line 14, in script
    tc "So I bet you're wondering where I was?"
IOError: Couldn't find file 'images/by_Sysen/Troy/barista1.png'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 614, in script call
    credits = ('Backgrounds', 'Uncle Mugen'), ('Sprites', 'Deji'), ('Writing', 'meltycreamcats'), ('Writing', 'margaretcatter'), ('Programming', 'margaretcatter'), ('Programming', 'Arsym'), ('GUI Design', 'Re:Alice') ('Music', 'Kevin MacLeod'), ('Music', 'XXX'), ('SFX', 'Mike Koenig'), ('GFX Wipes', 'Kia')
  File "game/Beta Ending.rpy", line 14, in script
    tc "So I bet you're wondering where I was?"
  File "L:\Programs\Renpy\renpy-7.2.2-sdk\renpy\ast.py", line 694, in execute
    renpy.exports.say(who, what, *args, **kwargs)
  File "L:\Programs\Renpy\renpy-7.2.2-sdk\renpy\exports.py", line 1322, in say
    who(what, *args, **kwargs)
  File "L:\Programs\Renpy\renpy-7.2.2-sdk\renpy\character.py", line 1127, in __call__
    self.do_display(who, what, cb_args=self.cb_args, **display_args)
  File "L:\Programs\Renpy\renpy-7.2.2-sdk\renpy\character.py", line 842, in do_display
    **display_args)
  File "L:\Programs\Renpy\renpy-7.2.2-sdk\renpy\character.py", line 591, in display_say
    rv = renpy.ui.interact(mouse='say', type=type, roll_forward=roll_forward)
  File "L:\Programs\Renpy\renpy-7.2.2-sdk\renpy\ui.py", line 289, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "L:\Programs\Renpy\renpy-7.2.2-sdk\renpy\display\core.py", line 2690, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, **kwargs)
  File "L:\Programs\Renpy\renpy-7.2.2-sdk\renpy\display\core.py", line 3174, in interact_core
    self.draw_screen(root_widget, fullscreen_video, (not fullscreen_video) or video_frame_drawn)
  File "L:\Programs\Renpy\renpy-7.2.2-sdk\renpy\display\core.py", line 2086, in draw_screen
    renpy.config.screen_height,
  File "render.pyx", line 516, in renpy.display.render.render_screen
  File "render.pyx", line 244, in renpy.display.render.render
  File "L:\Programs\Renpy\renpy-7.2.2-sdk\renpy\display\layout.py", line 722, in render
    surf = render(child, width, height, cst, cat)
  File "render.pyx", line 151, in renpy.display.render.render
  File "render.pyx", line 244, in renpy.display.render.render
  File "L:\Programs\Renpy\renpy-7.2.2-sdk\renpy\display\layout.py", line 722, in render
    surf = render(child, width, height, cst, cat)
  File "render.pyx", line 151, in renpy.display.render.render
  File "render.pyx", line 244, in renpy.display.render.render
  File "L:\Programs\Renpy\renpy-7.2.2-sdk\renpy\display\layout.py", line 722, in render
    surf = render(child, width, height, cst, cat)
  File "render.pyx", line 151, in renpy.display.render.render
  File "render.pyx", line 244, in renpy.display.render.render
  File "accelerator.pyx", line 110, in renpy.display.accelerator.transform_render
  File "render.pyx", line 244, in renpy.display.render.render
  File "L:\Programs\Renpy\renpy-7.2.2-sdk\renpy\display\image.py", line 500, in render
    return wrap_render(self.target, width, height, st, at)
  File "L:\Programs\Renpy\renpy-7.2.2-sdk\renpy\display\image.py", line 306, in wrap_render
    rend = render(child, w, h, st, at)
  File "render.pyx", line 151, in renpy.display.render.render
  File "render.pyx", line 244, in renpy.display.render.render
  File "L:\Programs\Renpy\renpy-7.2.2-sdk\renpy\display\layout.py", line 722, in render
    surf = render(child, width, height, cst, cat)
  File "render.pyx", line 151, in renpy.display.render.render
  File "render.pyx", line 244, in renpy.display.render.render
  File "L:\Programs\Renpy\renpy-7.2.2-sdk\renpy\display\im.py", line 580, in render
    return cache.get(self, render=True)
  File "L:\Programs\Renpy\renpy-7.2.2-sdk\renpy\display\im.py", line 266, in get
    surf = image.load()
  File "L:\Programs\Renpy\renpy-7.2.2-sdk\renpy\display\im.py", line 625, in load
    surf = renpy.display.pgrender.load_image(renpy.loader.load(self.filename), self.filename)
  File "L:\Programs\Renpy\renpy-7.2.2-sdk\renpy\loader.py", line 576, in load
    raise IOError("Couldn't find file '%s'." % name)
IOError: Couldn't find file 'images/by_Sysen/Troy/barista1.png'.

Windows-8-6.2.9200
Ren'Py 7.2.2.491
Hurricane Like Me Renpy 7.2.2.491
Mon Sep 16 23:34:54 2019
